Week6/Day3/CommunityShedMVC/CommunityShedMVC/Controllers/AccountController.cs

```diff
@@ -0,0 +1,92 @@
+﻿using CommunityShedMVC.Data;
+using CommunityShedMVC.Models;
+using CommunityShedMVC.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using System.Web.Security;
+
+namespace CommunityShedMVC.Controllers
+{
+    [Authorize]
+    public class AccountController : Controller
+    {
+        [AllowAnonymous]
+        public ActionResult Register()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        [AllowAnonymous]
+        public ActionResult Register(RegisterViewModel viewModel)
+        {
+            //Validate that the provided email isn't already in use
+
+            if (ModelState.IsValid)
+            {
+                string hashedPassword = BCrypt.Net.BCrypt.HashPassword(viewModel.Password, 12);
+                DatabaseHelper.Insert(@"
+                    insert into Person (
+                        FirstName,
+                        LastName,
+                        Email,
+                        Password)
```

Comment:
I'd recommend that you name this column `HashedPassword`. That'll help you and others on your team to distinguish between clear text and hashed passwords.